payroll-processor-client/src/app/data/data-service.ts

```diff
@@ -25,46 +23,32 @@ export class DataService {
 
   private loadData() {
     this.http
-      .get<Employee[]>(`${this.apiUrl}/EmployeesGetTrigger`)
+      .get<Payroll[]>(`${this.apiUrl}/Payrolls`)
```

Comment:
nit: all the urls in the functions endpoints are `dash-case`

Reply:
we'll have to keep this in mind if we expand beyond crud